### PR TITLE
[Component with its own class name as service type] LPS-199467 Refactor Helper and importer class from wiki module

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-199467

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:wiki

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/base/BaseWikiMVCActionCommand.java [hunk 1]
```diff
@@ -31,18 +32,18 @@
...
-import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 
 /**
- * @author Roberto Díaz
+ * @author Regisson Aguiar
  */
-@Component(service = WikiAttachmentsHelper.class)
-public class WikiAttachmentsHelper {
+public abstract class BaseWikiMVCActionCommand extends BaseMVCActionCommand {
```
Comment (rafaprax) [inline modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/base/BaseWikiMVCActionCommand.java:40]: Hey @MrJohn1911, rename to just `BaseMVCActionCommand `. Thx

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java [hunk 2]
```diff
@@ -570,6 +603,192 @@ private List<WikiNode> _addDefaultNode(long groupId)
 		return ListUtil.fromArray(node);
 	}
 
+	private String _getCreoleRedirectContent(String redirectTitle) {
```
Comment (rafaprax) [inline modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java:606]: He @MrJohn1911, in this case, we are mixing up the `MediaWikiImporter` and the `WikiNodeLocalService` responsibilities. So, keep it as a separate class and inject the dependencies via constructor. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
@MrJohn1911 fix and resend

**Pull request closed without merging**